lib/rex/ui/interactive.rb

```diff
@@ -83,8 +83,13 @@ def interact(user_input, user_output)
       self.completed = true
     end
 
-    # Return whether or not EOF was reached
-    return eof
```

Comment:
I couldn't find a place either. I see _interact checks if the session is stopped directly anyway.